### PR TITLE
Update co-authoring-overview for issue #961

### PR DESCRIPTION
**Pull request by ojrb**
Hi @scanum I modify the excel client co-authoring support for this article, based on the recommendations for issue #961, this PR solve this issue, Please review and Merge

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/governance/co-authoring-overview.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ When a user saves a Word, PowerPoint, or Word Online document, other current use
   
 In OneNote and OneNote Web App, shared notebooks enable users to share notes seamlessly. When a user edits a page of the notebook, those edits are automatically synchronized with other users of that notebook so that everybody has a complete set of notes. Edits made by multiple users on the same page appear automatically, which enables near real-time collaboration. Versioning and other shared features in OneNote make it possible for users to roll back edits, show what edits are new, and determine who made a specific edit. 
   
-The Excel client application does not support co-authoring workbooks in SharePoint Server. The Excel client application uses the Shared Workbook feature to support non-real-time co-authoring workbooks that are stored locally or on network (UNC) paths. Co-authoring workbooks in SharePoint is supported by using the Excel Web App. 
+The Excel client application now support co-authoring workbooks in SharePoint Server.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/governance/co-authoring-overview.md]:
```suggestion
The Excel client application now supports co-authoring workbooks in SharePoint Server. 
```

**Comment by ojrb** [comment]
Hi @scanum I committed the suggestions of @JohanFreelancer9 !

**Comment by scanum** [comment]
@tomresing Hi Tom, changes look correct. Could you please check and merge? Thanks!

**Comment by tomresing** [comment]
I don't think this has changed. The Excel writers have not updated[ their article on the same subject.](https://support.office.com/en-us/article/collaborate-on-excel-workbooks-at-the-same-time-with-co-authoring-7152aa8b-b791-414c-a3bb-3024e46fb104) I will double check.

**Comment by scanum** [comment]
@tomresing Any updates on this thanks!

**Comment by tomresing** [comment]
The rich client version of Excel on Windows doesn’t support co-authoring with any version of SharePoint Server. Some versions of Excel on Windows do support co-authoring with SharePoint Online.

**Pull request closed without merging**